### DISCUSSION TITLE
[Idea] Modules configuration page useful improvements

### DISCUSSION THREAD

**Discussion by Caleydon** [PrestaShop/PrestaShop#30972] [Ideas]
I'd like to share a few ideas (some of them I already hardcoded into my PS installation) for improvements modules configuration page.

### The solution

Modules configuration page useful improvements:
1. make breadcrumb active
2. add module icon and make module name as major information
3. add module status indicator (mainly when module is disabled)
4. allow enable/disable module from module configuration page
5. detailed module status - display an information if the module is enabled/disabled for monitors/mobile displays

![module-configuration-ideas](https://user-images.githubusercontent.com/2806752/213674984-56aefc75-4b87-4a2c-b6e2-543aab7a0503.png)

What do you think?

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by maniT4c** [comment]
I'm ok with all the suggestion, i think it's a good idea to improve this part !

**Comment by kpodemski** [comment]
Looks good! @Caleydon will you be interested in helping have it in the core? It's been some time since the last time you contributed to the core, I can help you get through the process :D

**Comment by MatShir** [comment]
I was thinking if we could also improve the module page or the banner with button on the top. I have noticed that each module has their own way to display contact form displaying phone or emails and other way to link their own documentation and so on. Furthermore, I was wondering with a more structured header, it will help the merchant to find the information always.
